Census API to Socrata/PythonCensusApiCityReader.py

```diff
@@ -0,0 +1,64 @@
+import fme
+import fmeobjects
+import json
+import urllib2
+from urllib2 import URLError
+
+# Construct URL for API call to Census
+# example (commute time): http://api.census.gov/data/2013/acs1?get=NAME,B08303_001E,B08303_002E,B08303_003E,B08303_004E,B08303_005E,B08303_006E,B08303_007E,B08303_008E,B08303_009E,B08303_010E,B08303_011E,B08303_012E,B08303_013E&for=place:63000&in=state:53
+
+class PythonCensusApiCityReader(object):
+    def __init__(self):
+        
+        self.apikey = fme.macroValues['API_KEY']
+        self.acsyear = fme.macroValues['ACS_YEAR']
+        self.indicators = fme.macroValues['INDICATORS']
+        self.url = fme.macroValues['URL']
+        self.years = fme.macroValues['YEARS']
+        self.cities = [fme.macroValues['CITY1'],fme.macroValues['CITY2'],fme.macroValues['CITY3'],fme.macroValues['CITY4'],fme.macroValues['CITY5'],fme.macroValues['CITY6'],fme.macroValues['CITY7'],fme.macroValues['CITY8']]
+
+        self.log = fmeobjects.FMELogFile()
+
+    def close(self):
+		# loop through years
+		lstYears = self.years.split(",")
```

Comment:
Nit-picking: For variable naming, we should be using lowercase with words separated by underscores as described in PEP 8 (related: http://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function-names).